### PR TITLE
Move docker commands for building owl files to a script

### PR DESCRIPTION
**Pull request by glguy**
this ought to make it easier for developers to generate and see the OWL files that will be generated by our CI process

### CODE REVIEW

**Comment by glguy** [comment]
I *think* this is a nearly cosmetic change, but it would be nice to have the commands used by CI to be maintained in a simple place for people to be able to run them outside of the CI process without having to check to see if the CI yaml file has changed or to consult that file to find the commands.

**Review by tuxji — APPROVED** [review]
+1 with suggestions for improvements

File: cli/setup-build-owl.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+REPO_DIR=$(dirname "$PWD")
```
[the remark this answers is not in the document]
Comment (glguy) [reply cli/setup-build-owl.sh]: docker wanted an aboslute path, so I used `realpath` to obtain one

**Comment by glguy** [comment]
@tuxji Are these stack traces that seem to appear when trying to delete something a thing we could fix with extra flags to docker? I'm wondering if it's in a mode where you aren't allowed to delete files.

```
Building RACK-Ontology: Deleting '/RACK-Ontology/OwlModels/HAZARD.owl'.
423
24444 [main] ERROR org.eclipse.xtext.builder.impl.XtextBuilder  - Problems encountered while deleting resources.
424
org.eclipse.core.internal.resources.ResourceException: Problems encountered while deleting resources.
```

**Comment by tuxji** [comment]
@glguy I'm not sure what's causing the deletion error, but my suspicion is that it's a fluke, not a reproducible error (this is the first time we've seen that error). 

You do have to fix another problem.  The lint job is failing since we need to put quotation marks around `$(dirname "$0")/..`:

```
In ./cli/setup-build-owl.sh line 6:
RACK_DIR=$(realpath $(dirname "$0")/..)
                    ^-------------^ SC2046 (warning): Quote this to prevent word splitting.
```

Once you push that fix, we'll see if the deletion error happens again.

**Comment by tuxji** [comment]
CI works, go ahead and merge this PR.

**Pull request merged**